proto/message.py

```diff
@@ -327,7 +327,13 @@ def deserialize(cls, payload: bytes) -> "Message":
         """
         return cls.wrap(cls.pb().FromString(payload))
 
-    def to_json(cls, instance, *, use_integers_for_enums=True) -> str:
+    def to_json(
+        cls,
+        instance,
+        *,
+        use_integers_for_enums=True,
+        including_default_value_fields=True
```

Comment:
@yon-mg Could you open a follow up PR to update the docstring below?

This will result in a change in behavior for someone already using this method Protobuf defaults to `False` for this option. https://googleapis.dev/python/protobuf/latest/google/protobuf/json_format.html#google.protobuf.json_format.MessageToJson

Reply:
I'm not sure I understand. I did not change the way `google.protobuf.json_format.MessageToJson` works. In fact this method `message.MessageMeta.to_json` should call `MessageToJson` in the same way as it did before if no changes are made to the additional parameter I added.

Reply:
Although I did neglect to update the docstring for this method which I will update shortly if that's what you mean.